step.sh

```diff
@@ -2,6 +2,11 @@
 
 set -e
 
+# Install gpg for MacOS devices
+if [ -x "$(command -v brew)" ] && ! [ -x "$(command -v gpg)" ]; then
+  brew install gpg
+fi
+
 # Download and verify Codecov uploader
 curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --import # One-time step
```

Comment:
a note that, to suppress a warning about integrity of the check, the instructions for integrity checking have changed slightly: 

```
curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --curl -Os https://uploader.codecov.io/latest/macos/codecov
curl -Os https://uploader.codecov.io/latest/macos/codecov.SHA256SUM
curl -Os https://uploader.codecov.io/latest/macos/codecov.SHA256SUM.sig
gpgv codecov.SHA256SUM.sig codecov.SHA256SUM


shasum -a 256 -c codecov.SHA256SUM
chmod +x codecov
./codecov -t ${CODECOV_TOKEN}
```

This isn't required for things to work, but provides a ''cleaner'' experience imo.